dbt_project.yml

```diff
@@ -3,7 +3,7 @@ version: '0.13.0'
 config-version: 2
 require-dbt-version: ">=1.5.0"
 
-profile: default
```

Comment:
@kuljisd can you remove your changes to this file? Local testing config shouldn't be pushed in a branch.